test/e2e/restart_framework.go

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Given we have "reboot" and "restart" tests, and this framework doesn't apply to either of them, might it be more clear to just keep this in the daemon_restart file, where it's used?